### PR TITLE
cleanup test code in upgrades and autoscaling pkg

### PR DESCRIPTION
**Pull request by carlory**
**What type of PR is this?**

/kind cleanup
/kind testing 

**What this PR does / why we need it**:

Use log functions of core e2e framework instead of e2elog sub package

```
test/e2e/autoscaling/*.go
test/e2e/upgrades/apps/daemonsets.go
test/e2e/upgrades/*.go
test/e2e/upgrades/storage/persistent_volumes.go
```

**Which issue(s) this PR fixes**:

refer #81427

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

**Review by krzyzacy — APPROVED** [review]

**Comment by carlory** [comment]
@krzyzacy Thanks for your approval. Can you give me a ok-to-test label on this pr?

**Comment by carlory** [comment]
ping @krzyzacy @carlory 

It' s ready for review. Could you take a quick look at it?

File: test/e2e/upgrades/apps/deployments.go [hunk 1]
```diff
@@ -26,9 +26,9 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2edeploy "k8s.io/kubernetes/test/e2e/framework/deployment"
 	"k8s.io/kubernetes/test/e2e/upgrades"
+	imageutils "k8s.io/kubernetes/test/utils/image"
 
 	"github.com/onsi/ginkgo"
-	imageutils "k8s.io/kubernetes/test/utils/image"
```
Comment (oomichi) [inline test/e2e/upgrades/apps/deployments.go]: Please don't change unrelated file.
This doesn't contain e2elog.

File: test/e2e/upgrades/apps/replicasets.go [hunk 2]
```diff
@@ -26,9 +26,9 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/replicaset"
 	"k8s.io/kubernetes/test/e2e/upgrades"
+	imageutils "k8s.io/kubernetes/test/utils/image"
 
 	"github.com/onsi/ginkgo"
-	imageutils "k8s.io/kubernetes/test/utils/image"
```
Comment (oomichi) [inline test/e2e/upgrades/apps/replicasets.go]: ditto

File: test/e2e/upgrades/apps/statefulset.go [hunk 3]
```diff
@@ -20,9 +20,8 @@ import (
 	"github.com/onsi/ginkgo"
 
 	appsv1 "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/version"
-
```
Comment (oomichi) [inline test/e2e/upgrades/apps/statefulset.go]: ditto

File: test/e2e/upgrades/sysctl.go [hunk 4]
```diff
@@ -26,7 +26,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/kubernetes/pkg/kubelet/sysctl"
-
```
Comment (oomichi) [inline test/e2e/upgrades/sysctl.go]: ditto

**Review by oomichi — CHANGES_REQUESTED** [review]
This also changed unrelated files.
Please keep the PR scope simple and clean for the PR message and the corresponding issue.

**Comment by carlory** [comment]
@oomichi 

It's ready for re-review.  Could you take a quick look at it?

**Comment by oomichi** [comment]
@carlory Thanks for updating, but unfortunately we need to rebase this patch again for the latest master?
Could you do that?

**Comment by carlory** [comment]
@oomichi  done!

**Review by oomichi — COMMENTED** [review]
/lgtm

**Pull request merged**